visualization/index.html

```diff
@@ -62,9 +73,10 @@
                 <h3>ASL-LEX 2.0</h3>
             </div>            
             <div class="col-10">
-                <button href="" class="btn btn-secondary" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <button href="" class="btn btn-secondary" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" style="background-color: #433089; border: #433089">
```

Comment:
New styles we add should be classes with corresponding definitions in the appropriate css file. I know we have other stray styles lying around, and we can't refactor everything, but we can do so for the new styles we add